src/types/api/auditlog.ts

```diff
@@ -0,0 +1,217 @@
+  /** object: guild;  change in number of days after which inactive and role-unassigned members are kicked */
+  prune_delete_days: number;
+  /** object: guild; server widget enabled/disable */
+  widget_enabled: boolean;
+  /** object: guild; channel id of the servere widget changed */
+  widget_channel_id: string;
+  /** object: guild; id of the system channel changed */
+  system_channel_id: string;
+  /** object: channel; text or voice channel position changed */
+  position: number;
+  /** object: channel; text channel topic changed */
+  topic: string;
+  /** object: channel; voice channel bitrate changed */
+  bitrate: number;
+  /** object: channel; permissions on a channel changed */
+  permission_overwrites: DiscordOverwrite[];
+  /** object: channel; channel nsfw restriction changed */
+  nsfw: boolean;
+  /** object: channel; application id of the added or removed webhook or bot */
+  application_id: string;
+  /** object: channel; amount of seconds a user has to wait before sending another message changed */
+  rate_limit_per_user: number;
+  /** object: role; permissions for a role changed */
+  permissions: string;
+  /** object: role; role color changed */
+  color: number;
+  /** object: role; role is now displayed/no longer displayed seperate from online users */
+  hoist: boolean;
+  /** object: role; rrole is now mentionable/unmentionable */
+  mentionable: boolean;
+  /** object: role; a permission on a text or voice channel was allowed for a role */
+  allow: string;
+  /** object: role; a permission on a text or voice channel was denied for a role */
+  deny: string;
+  /** object: invite; invite code changed */
+  code: string;
+  /** object: invite; channel for invite code changed */
+  channel_id: string;
+  /** object: invite; person who created invite code changed */
+  invite_id: string;
+  /** object: invite; change to max number of times invite code can be used */
+  max_uses: number;
+  /** object: invite; number of times invite code used changed */
+  uses: number;
+  /** object: invite; how long invite code lasts changed */
+  max_age: number;
+  /** object: invite; invite code is temporary/never expires */
+  temporary: boolean;
+  /** object: user; user server deafened/undeafened */
+  deaf: boolean;
+  /** object: user; user server muted/unmuted */
+  mute: boolean;
+  /** object: user; user nickanem changed */
+  nick: string;
+  /** object: user; user avatar changed */
+  avatar_hash: string;
+  /** object: any; the id of the changed entity - sometimes used in conjunction with other keys */
+  id: string;
+  /** object: any; type of entity created */
+  type: DiscordChannelTypes | string;
+  /** object: integration; integration emoticons enabled/disabled */
+  enable_emoticons: boolean;
+  /** object: integration; integration expiring subscriber behavior changed */
+  expire_behavior: number;
+  /** object: integration; integration expire grace period changed */
+  expire_grace_period: number;
+}
```

Comment:
This is not correct (https://github.com/discordeno/discordeno/pull/487/files)

Reply:
We should just remove the types that are covered in #487 already from this PR

Reply:
No we cannot to this since this are two different files

Reply:
Does not matter, this type is not used in production, so it should be fine

Reply:
This file here is in `src/types/api/auditlog.ts` and in #487 it is `src/types/guild.ts` so it does matter

Reply:
And?

Reply:
So we cannot remove the types from here and merge the other PR.

Reply:
Your reason?

Reply:
wdym

Reply:
What is the reason as to why this cannot be removed from this PR?

Reply:
As said #487 edits `src/types/guild.ts` but in this PR it is `src/types/api/auditlog.ts` This PR is here to improve the API types then having some audit log types in guild.ts again does not make sense